src/raftstore/store/keys.rs

```diff
@@ -199,6 +200,11 @@ pub fn origin_key(key: &[u8]) -> &[u8] {
     &key[DATA_PREFIX_KEY.len()..]
 }
 
+/// The caller should ensure the key is a timestamped key.
+pub fn truncate_ts(key: &[u8]) -> &[u8] {
```

Comment:
The function should be put in `types.rs`. All function in this file has nothing related to MVCC.